src/features/LandingPage/HeroHeader/index.tsx

```diff
@@ -41,6 +43,13 @@ const HeroHeader = () => {
     document.getElementById('__docusaurus').className = 'docs-landing-page';
   }
 
+  // To avoid weird colour issues, force Firefox to use webm, other browsers get mp4
+  const injectVideo = () => {
+    const isFirefox = ExecutionEnvironment.canUseDOM && navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
```

Comment:
```suggestion
    const isFirefox = ExecutionEnvironment.canUseDOM && navigator.userAgent.toLowerCase().includes('firefox');
```

Using `includes` is more intuitive

Reply:
Thanks Adrian!